packages/react-examples/src/react-charting/DonutChart/DonutChart.Multiple.Example.tsx

```diff
@@ -0,0 +1,150 @@
+import * as React from 'react';
+import {
+  DonutChart,
+  IDonutChartProps,
+  IChartProps,
+  IChartDataPoint,
+  DataVizPalette,
+  getColorFromToken,
+  getGradientFromToken,
+  DataVizGradientPalette,
+} from '@fluentui/react-charting';
+import { Toggle } from '@fluentui/react/lib/Toggle';
+
+interface IDonutChartState {
+  enableGradient: boolean;
+  roundCorners: boolean;
+}
+
+export class DonutChartMultipleExample extends React.Component<IDonutChartProps, IDonutChartState> {
```

Comment:
>DonutChartMultipleExample [](http://example.com/codeflow?start=13&length=25)

don't create a new example. Add a toggle in the existing example.

Reply:
done